src/06_tuples.py

```diff
@@ -37,6 +37,7 @@ def dist(a, b):
 def print_tuple(data):
     for i in range(1, len(data)):
         print(data[i])
+        
```

Comment:
This is a great use of `range` and `len` here!

Tuples are iterable objects in python just like lists, however. 

See if you can simplify this loop with that in mind.